docs/docs/gatsby-on-linux.md

```diff
@@ -2,12 +2,128 @@
 title: Gatsby on Linux
 ---
 
-> This is a TODO. Help our community expand it.
+# Linux
 
-> Please use the [Gatsby Style Guide](/contributing/gatsby-style-guide/) to ensure your pull request gets accepted.
+This guide assumes you already have a native installation of Linux on your machine. The following steps walk through how to install `node` and associated depdendencies.
+
+### Ubuntu, Debian and other `apt` based distros
+
+Begin by updating and upgrading.
+
+```shell
+sudo apt update
+sudo apt -y upgrade
+```
+
+> Only use `-y` if you want to upgrade to the latest version of the software.
```

Comment:
The `-y` iirc will just automatically move forward w/o asking user for confirmation.

Reply:
I stole that from below, but good point on fixing the wording and being clearer.